lib/pagerduty/http_transport.rb

```diff
@@ -35,6 +35,6 @@ def http
   end
 
   def transported?(response)
-    response.kind_of? Net::HTTPSuccess or response.kind_of? Net::HTTPRedirection
+    response.is_a?(Net::HTTPSuccess) || response.is_a?(Net::HTTPRedirection)
```

Comment:
They are affectively the same, Rubocop just enforces one to keep everything the same